libminifi/test/unit/TimeUtilTests.cpp

```diff
@@ -342,3 +342,15 @@ TEST_CASE("Parse RFC3339", "[parseRfc3339]") {
   CHECK_FALSE(parseRfc3339(" 2023-03-01T19:04:55Z"));
   CHECK_FALSE(parseRfc3339("2023-03-01"));
 }
+
+TEST_CASE("Test human readable parser") {
+  using utils::timeutils::humanReadableDuration;
+  CHECK(humanReadableDuration(1234567us) == "1.23s");
```

Comment:
Can you add tests for ms and µs outputs, too, please?

Reply:
good idea :+1: based on cppref microseconds can be either us or µs
https://en.cppreference.com/w/cpp/chrono/duration/operator_ltlt
https://github.com/apache/nifi-minifi-cpp/pull/1907/commits/01f94f2e4f7dc432c502db043fb5e676a3969bde